plugins/mcreator-core/features/vertical_anchor_above_bottom.json

```diff
@@ -0,0 +1,18 @@
+{
+  "args0": [
+    {
+      "type": "field_number",
+      "name": "value",
+      "value": 5,
+      "precision": 1
+    }
+  ],
+  "output": "VerticalAnchor",
+  "colour": "#47B37D",

```

Comment:
Why specific RGB color and not some similar variant from hue range? (e.g. 170)

Reply:
Leftover from when the blocks were part of a test plugin (and also it's a pretty color)